plugin.py

```diff
@@ -102,4 +90,11 @@ def on_start(self, window) -> bool:
         return True
 
     def on_initialized(self, client) -> None:
-        pass
\ No newline at end of file
+        client.on_request(
+            'vscode/content',
+            lambda params, request_id: self.handle_vscode_content(client, params, request_id))
+
+    def handle_vscode_content(self, client, uri, request_id):
+        name = uri.split('/')[-1]
+        schema_resource = ResourcePath('Packages', PACKAGE_NAME, 'schemas', '{}.json'.format(name))
```

Comment:
This is so far the thing I am the most excited about :)
I think that this could be improved in the future 
to allow other sublime plugins to create a schema for their settings.

Reply:
Definitively, we can load schema files from other projects. Although we will need file mapping too so we would need to store that configuration somewhere. VSCode does that - it collects schemas from all installed extensions and stuff, but it's all javascript so it can get those from `package.json`.